<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Task</class>
 <widget class="QWidget" name="Task">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>进程管理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="listProcessWidget"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="showProcessButton">
         <property name="text">
          <string>刷新显示进程</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="startNewBoutton">
         <property name="text">
          <string>运行新进程</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="killButton">
         <property name="text">
          <string>结束进程</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
